sdk/python/feast/loaders/file.py

```diff
@@ -128,16 +134,33 @@ def export_dataframe_to_local(
     if dir_path is None:
         dir_path = tempfile.mkdtemp()
 
-    file_name = _get_file_name()
+    file_name = _get_file_name(data_format)
     dest_path = f"{dir_path}/{file_name}"
 
     # Temporarily rename datetime column to event_timestamp. Ideally we would
     # force the schema with our avro writer instead.
     df.columns = ["event_timestamp" if col == "datetime" else col for col in df.columns]
 
     try:
+
         # Export dataset to file in local path
-        to_avro(df=df, file_path_or_buffer=dest_path)
+        if data_format == DataFormat.DATA_FORMAT_AVRO:
+            to_avro(df=df, file_path_or_buffer=dest_path)
+        elif data_format == DataFormat.DATA_FORMAT_CSV:
+            # TODO: Remove this hidden coupling to PostgreSQL "COPY"
+            # The following code orders columns alphabetically (with event_timestamp last). This allows the COPY
+            # method of PostgreSQL to map the CSV columns correctly when loading it
+            columns = sorted(df.columns)
+            columns.pop(columns.index("event_timestamp"))
+            df[columns + ["event_timestamp"]].to_csv(
+                dest_path,
+                sep="\t",
```

Comment:
comma delimiter should be used when uploading the entity_row, since the snowflake store uses the default delimiter sep=","